chore: 8.4.0 alpha release notes (#17892)
OpenTrons/opentrons-api: api/release-notes.md

By installing and using Opentrons software, you agree to the Opentrons End-User License Agreement (EULA). You can view the EULA at [opentrons.com/eula](https://opentrons.com/eula).

---
## Opentrons Robot Software Changes in 8.4.0

Welcome to the v8.4.0 release of the Opentrons robot software! This release includes updated liquid handling commands for Opentrons-verified liquid classes and other new features, improvements, and bug fixes. 

### New Features

> - Use new ``transfer_liquid``, ``distribute_liquid``, and ``consolidate_liquid`` commands only on the Flex to optimize liquid handling based on Opentrons-verified liquid classes. 

Reviewer: ```suggestion
- Use new ``transfer_liquid``, ``distribute_liquid``, and ``consolidate_liquid`` commands on Flex to optimize liquid handling based on Opentrons-verified liquid classes. 
```
I think this matches how we've described Flex-only features in the past.

> - Stack multiple Opentrons Tough auto-sealing lids on the deck. 
> - Move Opentrons Tough auto-sealing lids or remove tip rack lids with the Flex gripper. 

Reviewer: ```suggestion
- Stack multiple Opentrons Tough PCR Auto-Sealing Lids on the deck. 
- Move Opentrons Tough PCR Auto-Sealing Lids or remove tip rack lids with the Flex Gripper. 
```

- Aspirate or dispense in a well based on the liquid meniscus. 

### Improvements

> - Improvements to ``return_tip`` change the pipette position to correctly drop and return attached tips to the tip rack. 

Reviewer: Not sure what this refers to. I think we need something more precise.

Author: Refers to [EXEC-1191](https://opentrons.atlassian.net/browse/EXEC-1191), this static electricity issue. Probably need to touch base with Ryan on an improved description for this one.

Reviewer: Aha. It's a general improvement. Something like "Improved tip-drop motion for Flex 8-channel pipettes."

### Bug Fixes

> - An error is raised if the Absorbance Plate Reader fails to initialize before reading a plate inside the module. 

Reviewer: ```suggestion
- The API raises an error if the Absorbance Plate Reader fails to initialize before reading a plate inside the module. 
```
active because we can
